### PR TITLE
Feat/supabase

### PR DESCRIPTION
**Pull request by elrincondeldev**
- Se ha conectado el formulario de registro con supabase.

### CODE REVIEW

File: src/components/Register/index.tsx [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
     },
   ],
   terms_and_conditions: false,
-} as const
+} as const;
 
 export const Register = () => {
+  const supabaseUrl = import.meta.env.VITE_SUPABASE_URL;
+  const supabaseKey = import.meta.env.VITE_SUPABASE_KEY;
+  const supabase = createClient(supabaseUrl, supabaseKey);
```
Comment (MarcosNASA) [inline src/components/Register/index.tsx:28]: You should extract this outside of the component so it doesn't create a new client on every render.

Comment (afordigital) [reply src/components/Register/index.tsx:28]: I can't merge your PR until you resolve errors in deploy

File: .gitignore [hunk 2]
```diff
@@ -22,3 +22,6 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+# .env
```
Comment (kevinzunigacuellar) [inline .gitignore:26]: remove this duplicate

File: src/components/Register/index.tsx (hunk 1 above)
Comment (kevinzunigacuellar) [reply src/components/Register/index.tsx:28]: @MarcosNASA React framework issue

File: src/components/Register/index.tsx [hunk 3]
```diff
@@ -32,13 +37,34 @@ export const Register = () => {
...
+        participant_name: participant.participant_name,
+        participant_country: participant.participant_country,
+        participant_email: participant.participant_email,
+        project_name: data.project_name,
+        project_description: data.project_description,
+        project_url: data.project_url,
+      }));
+
+      const { data: participantsData, error: participantsError } =
+        await supabase.from("Participants").insert(participants);
```
Comment (kevinzunigacuellar) [inline src/components/Register/index.tsx:54]: Handle errors gracefully
```suggestion
        const {error, data} = await supabase.from("Participants").insert(participants);
        if (error) { // code here}
```

File: src/components/Register/index.tsx [hunk 4]
```diff
@@ -32,13 +37,34 @@ export const Register = () => {
...
+
+      if (participantsError) {
+        throw participantsError;
```
Comment (kevinzunigacuellar) [inline src/components/Register/index.tsx:57]: Esto nos va a romper la app

File: src/components/Register/index.tsx [hunk 5]
```diff
@@ -32,13 +37,34 @@ export const Register = () => {
...
+      }
+
+      console.log("Participants Data:", participantsData);
```
Comment (kevinzunigacuellar) [inline src/components/Register/index.tsx:60]: se ha colado este console

**Comment by jarrisondev** [comment]
waos

**Pull request closed without merging**